SU-HSDO/suhumsci
# SHS-5693: Implementation: New Vertical Card option

# READY FOR REVIEW

## Summary
Add new `Vertical Button Card` option in `Postcard`

## Need Review By (Date)
07/26

## Urgency
medium

## Steps to Test
1. Import configuration, compile the theme and clear cache for Colorful and Traditional. Tugboat URLS: https://hs-traditional-n2nypkdhaunsv8whsxsquwupgep0exsv.tugboatqa.com/test-postcards and https://hs-colorful-n2nypkdhaunsv8whsxsquwupgep0exsv.tugboatqa.com/test-postcards
2. On Colorful, create a `Flexible Page`
3. Add a single`Postcard` component with the `Vertical Button Card`
4. Add Collection Cards (for all the different variants: With `Raised Cards`, with `Uniform Height`, without `Raised Cards` and `Uniform Height`, with different number of items per row )
5. Add `PostCards` with the `Vertical Button Card` in collections
6. Check they all look similar to what's in [Figma Option A](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1336-195&t=p1N2GbaHgX7wS5RH-4)
7. Please check the following with the different color palettes:
    
    > Title color: Secondary
    >  Button style: Secondary button
    >  Card outline:  Tertiary-Highlight

8. Now, on Traditional repeat steps 2, 3 and 4
9. Check they all look similar to what's in [Figma Option D (the last one)
](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1433-111&t=p1N2GbaHgX7wS5RH-4)
10. Please check the following with the different color palettes:

    > Card outline: Secondary
    > Button text color: White
    > Button background: Secondary
    > Button hover background: Primary-dark

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207881512681051
